package.json

```diff
@@ -15,7 +15,7 @@
   },
   "studio": {
     "version": "0.0.44",
-    "devBranch": "next"
+    "devBranch": "fl_cloud_nlu"
```

Comment:
??

Reply:
if you clone this branch, it'll download the studio binary from here: https://botpress-dev-bins.s3.amazonaws.com/

so you should be able to both train and predict using cloud nlu.

The way we merge is:
1. I merge the `fl_cloud_nlu` into `next` on studio
2. I update this branch to refer to dev branch `next`
3. I merge this branch into `next` on botpress

Many thanks for your contribution to this forum,

I hope you find this answer helpful,

If you have any other questions, please ask directly to @botpressops 

Kind regards,

François Levasseur